src/lib/up-storage.ts

```diff
@@ -228,6 +227,11 @@ class ProxyStorage implements IProxy {
       agentOptions: this.agent_options,
     };
 
+    // GET requests should not have a body, otherwise the request might fail(return 413 status code)
+    if (method.toUpperCase() !== 'GET') {
```

Comment:
great you found a solution, also thanks for the comment, would you mind add some test, here you can star looking at `test/unit/modules/uplinks/up-storage.spec.ts`

Reply:
I'll fix the code format and add some unit tests.